Dockerfile

```diff
@@ -2,9 +2,21 @@ FROM python:3.11-bullseye
 
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y git
+
+RUN git init /app/repo
```

Comment:
`git clone --depth 1 https://github.com/OpenFn/docs.git /app/repo` seems to do the job but it clones the entire repository. But since this only pulls the latest commit, the size is reduced significantly.
On the other hand, using sparse checkout we can just pull the docs folder from the repo.

The sparse checkout seems to take more memory. So, I guess we can use git clone.
Thanks for the suggestion.